mycore-mods/src/main/java/org/mycore/mods/MCRMODSCommands.java

```diff
@@ -195,6 +199,14 @@ private static MCRDerivate createDerivate(MCRObjectID documentID, File fileDir)
             firstRegularFile.ifPresent(ifs::setMainDoc);
             derivate.getDerivate().setInternals(ifs);
         }
+        MCRConfiguration2.getString("MCR.MODS.Import.Derivate.Categories")
+            .map(MCRConfiguration2::splitValue)
+            .ifPresent(s -> {
+                s.map(MCRCategoryID::fromString)
+                    .forEach(categId -> derivate.getDerivate().getClassifications()
+                        .add(new MCRMetaClassification("classification", 0, null,
```

Comment:
How does that work? The `classId` parameter of `MCRMetaClassification` is always `null` and the entire configuration value chunk (`derivate_types:content`) is passed as the `categId`, which will eventually be used as the value for `MCRCategoryID#setId(String)` where it is checked against the pattern `[^:$\{\}]+`, i.e. it is not allowed to contain a colon. Shouldn't the configuration value chunk be split at the colon ant the parts be used as `classId` and `categId`?